test/testNestingDepth.py

```diff
@@ -0,0 +1,119 @@
+import unittest
+from .testHelpers import get_cpp_function_list
+
+
+class TestCppNestingDepth(unittest.TestCase):
+
+    def test_one_function_with_no_nd_condition(self):
+        result = get_cpp_function_list("int fun(){}")
+        self.assertEqual(0, result[0].max_nesting_depth)
+
+    def test_one_function_with_one_nd_condition(self):
+        result = get_cpp_function_list("int fun(){if(a){xx;}}")
+        self.assertEqual(1, result[0].max_nesting_depth)
+
+    def test_one_function_nd_with_question_mark(self):
+        result = get_cpp_function_list("int fun(){return (a)?b:c;}")
+        self.assertEqual(1, result[0].max_nesting_depth)
+
+    def test_one_function_nd_with_forever_loop(self):
```

Comment:
Hmm, @rakhimov 's comment makes me wonder if forever loop should be counted as one CCN...

    while (i < 10) {}

is the same as:

    for(;;) {if (i < 10) break;}

So

    for(;;){}

shouldn't be counted probably. What do you think?